cli/azd/pkg/project/service_target_containerapp.go

```diff
@@ -99,6 +100,14 @@ func (at *containerAppTarget) Deploy(ctx context.Context, azdCtx *environment.Az
 	}
 
 	parametersFile := azdCtx.BicepParametersFilePath(at.env.GetEnvName(), at.config.ModuleName)
+
+	// If the bicep uses nested modules ensure the full directory tree
+	// is created before copying the parameters file.
+	directoryPath := filepath.Dir(parametersFile)
+	if err := os.MkdirAll(directoryPath, 0750); err != nil {
```

Comment:
NIT: We use 755 elsewhere - and I feel that's more common - Are we concerned about world readability here?  I think that 755 matches what git uses.

Reply:
We have a constant defined for this, IIRC
![image](https://user-images.githubusercontent.com/24213737/179317667-8e805863-eb1d-4c1b-ad53-1b78b2c76839.png)

cmd/init.go

Reply:
Consolidated file modes across CLI